app/presenters/spree/feed_product_presenter.rb

```diff
@@ -0,0 +1,274 @@
+      raise SchemaError.new("link", @product) unless @product_url.present?
+      @product_url
+    end
+
+    # Gives the formatted price of shipping for the product
+    #
+    # @return [String] the configured base shipping price, or
+    #   the minimum shipping available for this product.
+    def shipping_price
+      @shipping_price ||=
+        if bsp = Rails.configuration.try(:base_shipping_price)
+          Spree::Money.new(bsp)
+        else
+          Spree::Money.new(
+            @product.shipping_category.shipping_methods
+            .flat_map(&:shipping_rates)
+            .sort_by(&:cost)
+            .first
+            .cost)
+        end
+      raise SchemaError unless @shipping_price.present?
+      @shipping_price.money.format(symbol: false, with_currency: true)
+    end
+
+    # @return [String] the product sku
+    def id
+      @id ||= @product.sku
+      raise SchemaError.new("id", @product) unless @id
+      @id
+    end
+
+    # @return [String] the product name
+    def title
+      @title ||= @product.name
+      raise SchemaError.new("title", @product) unless @title.present?
+      @title
+    end
+
+    # @return [String] the product description
+    def description
+      @description ||= @product.description
+      raise SchemaError.new("description", @product) unless @description.present?
+      @description
+    end
+
+    # Returns the configured basic condition for products, or
+    # this products condition via product.property.
+    #
+    # @return [String] the product condition.
+    def condition
+      @condition ||=
+        Rails.configuration.try(:base_product_condition) || @product.property('condition') || 'new'
+      raise SchemaError.new("condition", @product) unless @condition.present?
+      @condition
+    end
+
+    # Computes whether this product has a brand
+    # and either, a GTIN number or an MPN number.
+    #
+    # @return [String] `no`, `yes`
+    def identifier_exists
+      ( brand? && (gtin? || mpn?) ) ? 'yes' : 'no'
+    end
+
+    # Gives the mandatory URL of the image of the product.
+    #
+    # @return [String, nil] a URL of an image of the product
+    def image_link
+      @images ||= @product.images.any? ? @product.images : @product.variants.flat_map { |v| v.images }
+      raise SchemaError.new("image link", @product) unless @images.length > 0
+      @image_link ||= @images.first.attachment.url(:large)
+      @image_link
+    end
+
+    # Computes the availability status of the product
+    # @return [String] the availability status of the product.
+    #   One of `in stock`, `out of stock`.
+    def availability
+      @availability ||= @product.stock_items.any?(&:available?) ? 'in stock' : 'out of stock'
+      raise SchemaError.new("availability", @product) unless @availability.present?
+      @availability
+    end
+
+    # Returns the most frequently used tax rate for this item. If no tax rate
+    # has been applied to the variant, the first tax rate is chosen.
```

Comment:
Can we leave taxation out of the feed? I can imagine way too many scenarios in which this would break: 

- Stores which sell into multiple tax jurisdictions
- US Stores that use Avalara or TaxCloud
- Jurisdictions that have more than one tax rate on things (Canada)